include/hal/library/cryptlib/cryptlib_cert.h

```diff
@@ -112,8 +112,11 @@ extern bool libspdm_x509_get_issuer_name(const uint8_t *cert, size_t cert_size,
  * @param[out]     extension_data       Extension bytes.
  * @param[in, out] extension_data_size  Extension bytes size.
  *
- * @retval  true
- * @retval  false
+ * @return true   If the returned extension_data_size == 0, it means that cert and oid are valid, but the oid extension is not found;
+ *                If the returned extension_data_size != 0, it means that cert and oid are valid, and the oid extension is found;
+ * @return false  If the returned extension_data_size == 0, it means that cert or oid are invalid;
+ *                If the returned extension_data_size != 0, it means that cert and oid are valid, and the oid extension is found,
```

Comment:
In this case does `extension_data_size` return the size of the `extension_data`?

Reply:
Yes. The returned `extension_data_size` is the size of the `extension_data`. Thanks.